lookbook/docs/patterns/14-danger-confirmation-dialog.md.erb

```diff
@@ -1,8 +1,13 @@
-The DangerConfirmationDialog is used to get a second layer of confirmation from the user before they execute a potentially breaking, non-reversible action such as deletion.
+The DangerDialog is used to direct the user's attention towards a potentially destructive, non-reversible action such as deletion.
 
-The dial will inform the user that the action is not reversible and require one additional interaction before the user can confirm the deletion.
+There are two variants:
 
-The DangerConfirmationDialog is a sub-component of FeedbackDialog.
+1. **Danger Confirmation** requires an addition interaction from the user (i.e, checking a checkbox) in addition to a visual and text warning before they are able to confirm the action
+2. **Danger Warning** simply presents the user with a visual and text warning before they confirm the action
+
+The dialog aims to reduce cases of accidental deletion that may result from the browser "Are you sure?" confirmation dialog not sufficiently conveying the potential risks of a particular user action.
+
+The DangerDialog is a sub-component of FeedbackDialog.
```

Comment:
This might be true in terms of UX, but isn't the case in terms of implementation - i.e. `DangerDialog` does not inherit from `FeedbackDialog`.